mode2.lua

```diff
@@ -945,10 +961,11 @@ function courseplay:unload_combine(vehicle, dt)
 		end
 
 		-- avoid circling
-		local distToChange = 1
+    -- if we are closer than distToChange meters to the current waypoint, we switch our target to the next
```

Comment:
@ThomasGaertner , I changed all of these to 5 meters (will get rid of the ifs if it is ok). The 1 m default seemed to cause a lot of issues, circling, swaying left and right, etc.
Is there any drawback of using a higher value like 5 meters?